.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
```

Comment:
This repository uses renovate so the tool shouldn't try and add dependabot:

https://github.com/jenkinsci/azure-sdk-plugin/blob/master/.github/renovate.json

Reply:
Sure thing, thank you. 👍

Reply:
thanks FWIW more general feedback for the tool, especially if we will be running it automatically in the future